app/views/manage_assessments/assignments/new.html.erb

```diff
@@ -1,5 +1,45 @@
-<%= simple_form_for @assignment, url: manage_assessments_outcome_coverage_assignments_path do |form| %>
-  <%= form.input :name, label: t(".assignment") %>
-  <%= form.input :problem, label: t(".problem") %>
+<%= content_for :full_bleed do %>
+  <div class="madlib-header">
+    <h1 class="headline-narrow">
+      Add assignment
+    </h1>
+    <h2 class="headline-narrow-subhead">
+      <%= @assignment.outcome_coverage.coverage.course.number %>
```

Comment:
This can be made `assignment.outcome.course.name`. @derekprior, I am playing around in the console in an attempt to make this less brittle. Any ideas? I am happy to add some code in the model to accomplish this.